12.0.Dockerfile

```diff
@@ -0,0 +1,165 @@
+FROM python:3.7-stretch AS base
```

Comment:
This should be 3.5 for adjusting to Odoo. There are know incompatibilities with this latest version.l

Reply:
AFAIK from https://github.com/odoo/odoo/pull/26587 odoo 11 is compatible already. Are you sure master is not?

Reply:
I prefer to stick to exact compatible version in this case. No real advantages of using a later version, as we can't use new idiomatics. Is there any performance advantage? And why use pinned module versions, but not pin this?

Reply:
Pinning versions gives stability, while upgrading usually gives new features and better performance. That's why, as a general criteria, it's a good idea to upgrade dependencies before doing a new release, and freeze them once done.

However, I don't have benchmarks of Odoo on python 3.7 vs 3.5, if that's your question.

BTW, you didn't answer my question: are you sure odoo 12 is not compatible with python 3.7?

Reply:
> are you sure odoo 12 is not compatible with python 3.7?

No, but Odoo neither. My point is that we shouldn't risk for not known advantages. Even with advantages, we have a serious drawback: we will be able to develop in our stack things that are incompatible with the rest of Odoo because we don't remember that are 3.6 or 3.7 features, and as it works in our development environment...

Reply:
OK